edgehog-device-runtime-forwarder/Cargo.toml

```diff
@@ -26,3 +26,22 @@ rust-version = { workspace = true }
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+astarte-device-sdk = { workspace = true }
+backoff = { workspace = true, features = ["tokio"] }
+displaydoc = { workspace = true }
+edgehog-device-forwarder-proto = { workspace = true }
+futures = { workspace = true }
+hex = { workspace = true }
+http = { workspace = true }
+reqwest = { workspace = true }
+thiserror = { workspace = true }
+tokio = { workspace = true }
+tokio-tungstenite = { workspace = true }
+tracing = { workspace = true, features = ["log"] }
+tungstenite = { workspace = true }
```

Comment:
This can also be removed

Reply:
Apart from `tungstenite`, the other deps are necessary